doc/reference_processing.xml

```diff
@@ -1,6 +1,67 @@
 <?xml version="1.0" encoding="UTF-8"?>
 	<sect1 id="Geometry_Processing">
 		<title>Geometry Processing</title>
+
+	<refentry id="ST_Accum">
+	  <refnamediv>
+		<refname>ST_Accum</refname>
+
+		<refpurpose>An aggregate function to construct an array of geometries.</refpurpose>
```

Comment:
ST_Accum is effectively array_agg. I understand it was added when array_agg wasn't available, but how about removing it now that there is no supported Postgres versions without array_agg?

Reply:
Sounds good to me.  Can we get a third vote of +1?